### DISCUSSION TITLE
SynEditCb

### DISCUSSION THREAD

**Discussion by dm-e** [TurboPack/SynEdit#182] [Show and tell]
For users of C++Builder there is now a pure C++ project of the SynEdtor components:

[SynEditCb](https://github.com/dm-e/SynEditCb)

It is an experimental hardly tested project by which I like to test my Delphi2CB translation tool. For users of the C++Builder, however, it already has a major advantage. This is because C++Builder users cannot compile the demos unless they also have a Delphi license. With SynEditCb, the demos can be tried with the C++Builder even without installing the components

The project is based on the components before using DirectWrite. My attempts with the current version of the components failed because I cannot properly debug the Delphi code with the C++Builder. That would be necessary to remove some bugs.

My attempts to help here with the solution of the C++Builder problems also failed. See #153

**Comment by pyscripter** [comment]
Good to know about your translation to C++, but for this project we need to make the Delphi code usable in C++ Builder.  See my response in #153.
